src/Arcus.Scripting.DevOps/Scripts/Set-AzDevOpsArmOutputs.ps1

```diff
@@ -1,21 +1,19 @@
 param(
-   [string][parameter(Mandatory = $false)] $VariableGroupName,
-   [string][parameter(Mandatory = $false)] $ArmOutputsEnvironmentVariableName = "ArmOutputs",
-   [switch][parameter(Mandatory = $false)] $UpdateVariableGroup = $false,
-   [switch][parameter(Mandatory = $false)] $UpdateVariablesForCurrentJob = $false
+    [string][parameter(Mandatory = $false)] $VariableGroupName,
+    [string][parameter(Mandatory = $false)] $ArmOutputsEnvironmentVariableName = "ArmOutputs",
+    [switch][parameter(Mandatory = $false)] $UpdateVariableGroup = $false,
+    [switch][parameter(Mandatory = $false)] $UpdateVariablesForCurrentJob = $false
 )
 
-function Add-VariableGroupVariable()
-{
+function Add-VariableGroupVariable() {
     [CmdletBinding()]
     param
     (
```

Comment:
Should also be on the same line, then? Like `param(`

Reply:
Good one, fixed in [`901d049` (#424)](https://github.com/arcus-azure/arcus.scripting/pull/424/commits/901d0490716209e4a3a8a143f1037673457ca6bd)